docs/configuration/flagd_start.md

```diff
@@ -20,7 +20,8 @@ flagd start [flags]
   -k, --server-key-path string              Server side tls key path
   -d, --socket-path string                  Flagd socket path. With grpc the service will become available on this address. With http(s) the grpc-gateway proxy will use this address internally.
   -y, --sync-provider string                DEPRECATED: Set a sync provider e.g. filepath or remote
-  -a, --sync-provider-args stringToString   Sync provider arguments as key values separated by = (default [])
+  -a, --sync-provider-args stringToString   DEPRECATED: Sync provider arguments as key values separated by = (default [])
+  -s, --sync-providers string               JSON representation of an array of ProviderConfig objects. This object contains 2 required fields, uri (string) and provider (string). Documentation for this object can be found here: https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md#sync-provider-customisation
```

Comment:
I could go either way, but I'm wondering if `sources` is better language for this. I think it comes down to if we want to think of `sync-providers` as more of an internal detail to flagd or not. Currently, I think we use `sources` language in OFO. I'm interested in everyone's thoughts.

Reply:
ive introduced this change